### ISSUE TITLE
Block, Mute and Report

### ISSUE THREAD

**Issue by IngwiePhoenix** [PrimalHQ/primal-web-app#11]
## Summary
I would like to be able to block or mute certain spam accounts so they no longer show up in my timeline and potentially even put in reports.

Related NIPs:
- Reporting: https://github.com/nostr-protocol/nips/blob/master/56.md
- Lists: https://github.com/nostr-protocol/nips/blob/master/51.md

### Basic example
Add a drop-down menu for "More actions" within the row of Comment, [Zap,} Like, Repost to allow the user to select more options. This could alternatively be added in the top-right three-dot menu (Copy Nostr link, Copy Primal link) as well.

### Motivation
Right now, I do all the muting on Snort and Amethyst, but when I use Snort, my PC turns into a jet engine due to all the fetching activity happening on my client. Which isn't a bad idea, but it's a little bit like mining by trying to run `JSON.parse()` as fast and often as my hardware allows it. x) It would be neat to have this functionality in Primal, as I like the overall interface a lot more.

Thank you and kind regards,
Ingwie

**Comment by dwigton** [comment]
Agree. I really like Primal's way of filtering content. Explore -> Network is especially helpful, but I do come across explicit content that I'd rather not see. It would be nice to be able to block those accounts from the triple dot menu. It would also be nice to see which of my follows followed the account. If one account I follow repeatedly follows a bunch of objectionable accounts it would be nice to just drop them from recommending content. I'd still like to see their notes, just not the notes of those they follow.

**Comment by Shinoa-Fores** [comment]
Muting seems to work somewhat in the `development` branch, though it doesn't work 100% of the time. When I mute an account, one should not see that account show up in reposts or in any of the sidebars.

**Comment by IngwiePhoenix** [comment]
I originally wanted to close the issue since the two features are now implemented - however, it seems that they are not complete. 

Should I change the title to reflect that, perhaps?
